src/main/java/org/spdx/maven/utils/SpdxProjectInformation.java

```diff
@@ -85,9 +85,17 @@ public void setPrimaryPurpose( Purpose primaryPurpose )
         this.primaryPurpose = primaryPurpose;
     }
 
-    public SpdxProjectInformation () throws InvalidSPDXAnalysisException {
-        this.concludedLicense = new SpdxNoAssertionLicense();
-        this.declaredLicense = new SpdxNoAssertionLicense();
+    public SpdxProjectInformation ()
+    {
+        try
+        {
+            this.concludedLicense = new SpdxNoAssertionLicense();
+            this.declaredLicense = new SpdxNoAssertionLicense();
+        }
+        catch ( InvalidSPDXAnalysisException isae )
+        {
+            // ignore: cannot happen, why does SpdxNoAssertionLicense() constructor throw this exception?
```

Comment:
It should not happen, but it can in very unusual circumstances.  The SPDX Java Library supports a variety of stores through a storage interface.  Since there are calls to the underlying stores even for an `SpdxNoAssertionLicense()` it is possible that the store throws an exception which is translated to an `InvalidSPDXAnalysisException`.

In this plugin, we control the store and I don't think it will ever through an error in this case, but the interface still need to support the exception.

Rather than swallowing the exception, perhaps we can throw a runtime exception?

Reply:
getting such exceptions on basic `new SpdxNoAssertionLicense();` default license reference looks really weird and costs unexpected complexity for such default value

I'll revert this part, as there is a global approach to find, and I don't yet understand the details: this is not crucial for now

perhaps having a reusable constant for this default license would make sense

let's see that later